### PR TITLE
Fixed the documentation tree highlighting issue for trailing slashes

### PR DESCRIPTION
**Pull request by ruchikabgosain**
Bugzilla Bug: [XXXXX](https://bugzilla.mozilla.org/show_bug.cgi?id=XXXXX)

### CODE REVIEW

File: ui/src/components/Dashboard/DocsSidebarList.jsx [hunk 1]
```diff
@@ -142,8 +142,7 @@ export default class DocsSidebarList extends Component {
         <Typography component="span">(</Typography>
         {nodes.map((node, idx) => {
           const href = removeReadmeFromPath(`${DOCS_PATH_PREFIX}/${node.path}`);
-          const isLinkActive = removeReadmeFromPath(location.pathname) === href;
-
+          const isLinkActive = removeReadmeFromPath(location.pathname) === href || removeReadmeFromPath(location.pathname) === href + "/";
```
Comment (arku) [inline ui/src/components/Dashboard/DocsSidebarList.jsx]: @ruchikabgosain This should work but we can simplify it by removing the trailing slashes from the path.

```diff
--- a/ui/src/utils/removeReadmeFromPath.js
+++ b/ui/src/utils/removeReadmeFromPath.js
@@ -1 +1,2 @@
-export default (path = '') => path.replace(/\/?README\/?/, '');
+export default (path = '') =>
+  path.replace(/\/+$/, '').replace(/\/?README\/?/, '');
```

What do you think?

Comment (helfi92) [reply ui/src/components/Dashboard/DocsSidebarList.jsx]: I think that makes a lot of sense :)

Comment (ruchikabgosain) [reply ui/src/components/Dashboard/DocsSidebarList.jsx]: Hey @arku , I suppose you want me to directly update the <removeReadmeFromPath.js> and restore the changes for this <DocsSidebarList.jsx> file? 
I have updated it according to your changes, and i understand how this offers more clarity. 
Thankyou for your inputs :) Hope this resolves!

**Comment by ruchikabgosain** [comment]
Hey @helfi92, when i am doing `yarn lint --fix` it's not fixing anything and also yarn lint is not giving any warnings!

**Comment by helfi92** [comment]
You would need to do it inside the `ui` directory.

**Comment by ruchikabgosain** [comment]
Thankyou @helfi92 @arku for your time and helping me with my first PR to this project and especially mentioning @championshuttler for helping me out with this one!

**Pull request merged**